core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -90,37 +90,44 @@ object ConsumerGroupCommand {
 
     protected def opts: ConsumerGroupCommandOptions
 
-    protected def getLogEndOffset(topic: String, partition: Int): LogEndOffsetResult
+    protected def getLogEndOffset(topic: Option[String], partition: Option[Int]): LogEndOffsetResult
 
     protected def describeGroup(group: String): Unit
 
     protected def describeTopicPartition(group: String,
+                                         topic: String,
                                          topicPartitions: Seq[TopicAndPartition],
                                          getPartitionOffset: TopicAndPartition => Option[Long],
-                                         getOwner: TopicAndPartition => Option[String]): Unit = {
-      topicPartitions
-        .sortBy { case topicPartition => topicPartition.partition }
-        .foreach { topicPartition =>
-          describePartition(group, topicPartition.topic, topicPartition.partition, getPartitionOffset(topicPartition),
-            getOwner(topicPartition))
-        }
+                                         getOwner: TopicAndPartition => Option[String],
+                                         getMemberId: TopicAndPartition => Option[String]): Unit = {
+      if (topicPartitions.isEmpty)
+        describePartition(group, Some(topic), None, None, getOwner(null), getMemberId(null))
```

Comment:
In this case, since the member has no associated partitions, do we need to pass in topic at all? Could we just pass in None?